main-app/app/scripts/Proxy/proxy.js

```diff
@@ -0,0 +1,45 @@
+(function () {
+    'use strict';
+    angular.module('Tombola.BingoClient.Proxy')
+        .service('proxy',['$http', '$q', function ($http, $q) {
+
+            var me = this;
+
+            var callApi = function (endpoint, action, data, token){
```

Comment:
This is OK for now, but you might want to move this method out into its own service later if it is re-used.